### PR TITLE
Update main Win10 CI machine to RS4

### PR DESCRIPTION
**Pull request by danmosemsft**
Per https://github.com/dotnet/core-eng/issues/3420 this is now available.

@dotnet-bot test ci please

### CODE REVIEW

**Comment by danmosemsft** [comment]
@stephentoub interestingly most of the UnixDomainSocketTests fail on this RS4 box - with "Invalid argument" -- from the OS (it's in Spanish). Don't know if the Spanish part is relevant.

**Comment by danmosemsft** [comment]
https://mc.dot.net/#/user/danmosemsft/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/2153c3e3802f003414fd83740b228ad865f5c67b/workItem/System.Net.Sockets.Tests 
```
System.Net.Sockets.SocketException : Se ha proporcionado un argumento no válido
Stack Trace :
   at System.Net.Sockets.Socket.UpdateStatusAfterSocketErrorAndThrowException(SocketError error, String callerName) in D:\j\workspace\windows-TGrou---f8ac6754\src\System.Net.Sockets\src\System\Net\Sockets\Socket.cs:line 5157
   at System.Net.Sockets.Socket.DoBind(EndPoint endPointSnapshot, SocketAddress socketAddress) in D:\j\workspace\windows-TGrou---f8ac6754\src\System.Net.Sockets\src\System\Net\Sockets\Socket.cs:line 761
   at System.Net.Sockets.Socket.Bind(EndPoint localEP) in D:\j\workspace\windows-TGrou---f8ac6754\src\System.Net.Sockets\src\System\Net\Sockets\Socket.cs:line 702
   at System.Net.Sockets.Tests.UnixDomainSocketTest.Socket_SendReceiveAsync_PropagateToStream_Success(Int32 iterations, Int32 writeBufferSize, Int32 readBufferSize) in D:\j\workspace\windows-TGrou---f8ac6754\src\System.Net.Sockets\tests\FunctionalTests\UnixDomainSocketTest.netcoreapp.cs:line 200
--- End of stack trace from previous location where exception was thrown ---
```

**Comment by danmosemsft** [comment]
I hope they didn't break these domain sockets in RS4. I'll try ENU

**Comment by danmosemsft** [comment]
@dotnet/dnceng is there a US version of Windows.10.Amd64.ClientRS4.ES.Open ? Note it's RS4.

**Comment by danmosemsft** [comment]
Drawing tests failed with
```
2018-05-04 20:32:06,413: INFO: proc(54): run_and_log_output: Output: Starting:    System.Drawing.Common.Tests
2018-05-04 20:32:17,471: INFO: proc(54): run_and_log_output: Output:    System.Drawing.Printing.Tests.PrinterSettingsTests.CreateMeasurementGraphics_PageSettings_ReturnsExpected [FAIL]
2018-05-04 20:32:17,471: INFO: proc(54): run_and_log_output: Output:       System.OutOfMemoryException : Out of memory.
2018-05-04 20:32:17,471: INFO: proc(54): run_and_log_output: Output:       Stack Trace:
2018-05-04 20:32:17,471: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Graphics.Windows.cs(119,0): at System.Drawing.Graphics.FromHdcInternal(IntPtr hdc)
2018-05-04 20:32:17,471: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PrinterSettings.Windows.cs(631,0): at System.Drawing.Printing.PrinterSettings.CreateMeasurementGraphics(PageSettings pageSettings)
2018-05-04 20:32:17,471: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\tests\Printing\PrinterSettingsTests.cs(505,0): at System.Drawing.Printing.Tests.PrinterSettingsTests.CreateMeasurementGraphics_PageSettings_ReturnsExpected()

```

**Comment by MattGal** [comment]
Yes, windows.10.amd64.clientrs4.open

**Comment by MattGal** [comment]
@stephentoub  it is:
```
17134.1.x86fre.rs4_release.180410-1804
```
On both the en-us and es-es queues, which I believe is the proper release-to-world version, yes.

**Comment by stephentoub** [comment]
One thought: @MattGal or @danmosemsft, do we know what Path.GetTempPath() returns on these machines?  The Bind uses a file system path, and I'm wondering if maybe it's really long or something.

**Comment by danmosemsft** [comment]
Let's log temp.

**Comment by stephentoub** [comment]
@dotnet-bot test this please

**Comment by stephentoub** [comment]
@dotnet-bot test UWP CoreCLR x64 Debug Build please

**Comment by mmitche** [comment]
@dotnet-bot test UWP CoreCLR x64 Debug Build please

**Comment by stephentoub** [comment]
@dotnet-bot test Windows x86 Release Build please (merged logging updates)

**Comment by stephentoub** [comment]
@danmosemsft, I confirmed with @sunilmut that there's a current Windows limitation with Unix domain sockets in 32-bit processes on 64-bit OSes.  I've worked around that in the tests at https://github.com/dotnet/corefx/pull/29556.

**Comment by stephentoub** [comment]
@dotnet-bot test Windows x86 Release Build please (sockets issues should be fixed)

**Comment by stephentoub** [comment]
Failures are now all in System.Drawing.Common:
```
2018-05-08 11:58:49,451: INFO: proc(54): run_and_log_output: Output: Discovering: System.Drawing.Common.Tests
2018-05-08 11:58:50,562: INFO: proc(54): run_and_log_output: Output: Discovered:  System.Drawing.Common.Tests
2018-05-08 11:58:51,046: INFO: proc(54): run_and_log_output: Output: Starting:    System.Drawing.Common.Tests
2018-05-08 11:58:55,640: INFO: proc(54): run_and_log_output: Output:    System.Drawing.Printing.Tests.PageSettingsTests.Clone_Success [FAIL]
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:       System.ComponentModel.Win32Exception : Error en la llamada a procedimiento remoto
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:       Stack Trace:
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PrinterSettings.Windows.cs(896,0): at System.Drawing.Printing.PrinterSettings.GetHdevmodeInternal(String printer)
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PageSettings.Windows.cs(384,0): at System.Drawing.Printing.PageSettings.get_ExtraBytes()
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PageSettings.Windows.cs(368,0): at System.Drawing.Printing.PageSettings.CopyToHdevmode(IntPtr hdevmode)
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PageSettings.Windows.cs(235,0): at System.Drawing.Printing.PageSettings.get_PrinterResolution()
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\tests\Printing\PageSettingsTests.cs(41,0): at System.Drawing.Printing.Tests.PageSettingsTests.Clone_Success()
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:    System.Drawing.Printing.Tests.PrinterSettingsTests.CreateMeasurementGraphics_PageSettings_ReturnsExpected [FAIL]
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:       System.OutOfMemoryException : Out of memory.
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:       Stack Trace:
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Graphics.Windows.cs(119,0): at System.Drawing.Graphics.FromHdcInternal(IntPtr hdc)
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PrinterSettings.Windows.cs(631,0): at System.Drawing.Printing.PrinterSettings.CreateMeasurementGraphics(PageSettings pageSettings)
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\tests\Printing\PrinterSettingsTests.cs(505,0): at System.Drawing.Printing.Tests.PrinterSettingsTests.CreateMeasurementGraphics_PageSettings_ReturnsExpected()
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:    System.Drawing.Printing.Tests.PrintDocumentTests.BeginPrint_SetValue_ReturnsExpected [FAIL]
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:       System.ArgumentNullException : Value cannot be null.
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:       Parameter name: ptr
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:       Stack Trace:
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:             at System.Runtime.InteropServices.Marshal.StructureToPtr(Object structure, IntPtr ptr, Boolean fDeleteOld)
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PageSettings.Windows.cs(361,0): at System.Drawing.Printing.PageSettings.CopyToHdevmode(IntPtr hdevmode)
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PrintController.Windows.cs(276,0): at System.Drawing.Printing.PrintController.OnStartPrint(PrintDocument document, PrintEventArgs e)
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\tests\Printing\PrintDocumentTests.cs(275,0): at System.Drawing.Printing.Tests.PrintDocumentTests.TestPrintController.OnStartPrint(PrintDocument document, PrintEventArgs e)
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PrintController.Windows.cs(117,0): at System.Drawing.Printing.PrintController.Print(PrintDocument document)
2018-05-08 11:58:55,654: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\tests\Printing\PrintDocumentTests.cs(167,0): at System.Drawing.Printing.Tests.PrintDocumentTests.BeginPrint_SetValue_ReturnsExpected()
2018-05-08 11:58:57,641: INFO: proc(54): run_and_log_output: Output:    System.Drawing.Printing.Tests.PrinterSettingsTests.CreateMeasurementGraphics_Default_ReturnsExpected [FAIL]
2018-05-08 11:58:57,641: INFO: proc(54): run_and_log_output: Output:       System.OutOfMemoryException : Out of memory.
2018-05-08 11:58:57,641: INFO: proc(54): run_and_log_output: Output:       Stack Trace:
2018-05-08 11:58:57,641: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Graphics.Windows.cs(119,0): at System.Drawing.Graphics.FromHdcInternal(IntPtr hdc)
2018-05-08 11:58:57,641: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PrinterSettings.Windows.cs(631,0): at System.Drawing.Printing.PrinterSettings.CreateMeasurementGraphics(PageSettings pageSettings)
2018-05-08 11:58:57,641: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\tests\Printing\PrinterSettingsTests.cs(476,0): at System.Drawing.Printing.Tests.PrinterSettingsTests.CreateMeasurementGraphics_Default_ReturnsExpected()
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:    System.Drawing.Printing.Tests.PrinterSettingsTests.CreateMeasurementGraphics_Bool_ReturnsExpected [FAIL]
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:       System.DivideByZeroException : Attempted to divide by zero.
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:       Stack Trace:
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PageSettings.Windows.cs(111,0): at System.Drawing.Printing.PageSettings.get_HardMarginY()
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PrinterSettings.Windows.cs(622,0): at System.Drawing.Printing.PrinterSettings.CreateMeasurementGraphics(Boolean honorOriginAtMargins)
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\tests\Printing\PrinterSettingsTests.cs(491,0): at System.Drawing.Printing.Tests.PrinterSettingsTests.CreateMeasurementGraphics_Bool_ReturnsExpected()
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:    System.Drawing.Printing.Tests.PrintDocumentTests.QueryPageSettings_SetValue_ReturnsExpected [FAIL]
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:       System.NullReferenceException : Object reference not set to an instance of an object.
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:       Stack Trace:
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PageSettings.Windows.cs(388,0): at System.Drawing.Printing.PageSettings.get_ExtraBytes()
2018-05-08 11:58:59,346: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PageSettings.Windows.cs(368,0): at System.Drawing.Printing.PageSettings.CopyToHdevmode(IntPtr hdevmode)
2018-05-08 11:58:59,362: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PrintController.Windows.cs(276,0): at System.Drawing.Printing.PrintController.OnStartPrint(PrintDocument document, PrintEventArgs e)
2018-05-08 11:58:59,362: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\tests\Printing\PrintDocumentTests.cs(275,0): at System.Drawing.Printing.Tests.PrintDocumentTests.TestPrintController.OnStartPrint(PrintDocument document, PrintEventArgs e)
2018-05-08 11:58:59,362: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\src\System\Drawing\Printing\PrintController.Windows.cs(117,0): at System.Drawing.Printing.PrintController.Print(PrintDocument document)
2018-05-08 11:58:59,362: INFO: proc(54): run_and_log_output: Output:          D:\j\workspace\windows-TGrou---f8ac6754\src\System.Drawing.Common\tests\Printing\PrintDocumentTests.cs(230,0): at System.Drawing.Printing.Tests.PrintDocumentTests.QueryPageSettings_SetValue_ReturnsExpected()
```

**Comment by stephentoub** [comment]
@dotnet-bot test Outerloop Windows x86 Debug Build please

**Comment by danmosemsft** [comment]
@safern for drawing failures

**Comment by safern** [comment]
> @safern for drawing failures

Taking a look.

Also, should we use this PR to update official build queues to use RS4 queue?

**Comment by safern** [comment]
> @safern feel free to take this pr for that purpose if you want. Might be better to keep separate though.

Ok, let's keep it separate.

I've tried to repro the System.Drawing failures and I'm not able to reproduce them. I've used the CI machines snapshot with RS4 in spanish and the only failure that I get is: 

```
System.Drawing.Tests.BitmapTests.MakeTransparent_GrayscalePixelFormat_ThrowsArgumentException [FAIL]
      Assert.Throws() Failure
      Expected: typeof(System.Runtime.InteropServices.ExternalException)
      Actual:   (No exception was thrown)
      Stack Trace:
         E:\repos\corefxCopy\corefx\src\System.Drawing.Common\tests\BitmapTests.cs(964,0): at System.Drawing.Tests.BitmapTests.MakeTransparent_GrayscalePixelFormat_ThrowsArgumentException()
```

Which we've seen before in netfx: https://github.com/dotnet/corefx/issues/21886

It is worth saying that it fails consistently in all the machines with RS4 that I've tried to run the drawing tests, maybe GDI+ changed? Maybe a bug exposed in RS4 in [Graphics::DrawImage](https://msdn.microsoft.com/en-us/library/windows/desktop/ms536047(v=vs.85).aspx) code path?

I will re queue the builds and see what we get. I wonder why we're getting the other failures and in the 4 different machines I've tried it I'm not able to repro them.

**Comment by safern** [comment]
test Outerloop Windows x86 Debug Build please
test Windows x86 Release Build please

**Comment by safern** [comment]
test Outerloop Windows x86 Debug Build please
test Windows x86 Release Build please

restarting to get logs again and repro machines

**Comment by danmosemsft** [comment]
@safern I'm going to assign this to you as you own the next actions 😀

**Comment by safern** [comment]
test Outerloop Windows x86 Debug Build please

**Comment by stephentoub** [comment]
@dotnet-bot test this please

**Comment by stephentoub** [comment]
@dotnet-bot test Outerloop Windows x86 Debug Build please

**Comment by safern** [comment]
@dotnet/dnceng just FYI the Linux build seems to have succeeded but has error:

```
Using context: Linux arm64 Release Build
Could not update commit status of the Pull Request on GitHub.
org.kohsuke.github.HttpException: {
  "message": "Server Error"
}

	at org.kohsuke.github.Requester.handleApiError(Requester.java:690)
	at org.kohsuke.github.Requester._to(Requester.java:293)
	at org.kohsuke.github.Requester.to(Requester.java:234)
	at org.kohsuke.github.GHRepository.createCommitStatus(GHRepository.java:1075)
	at org.jenkinsci.plugins.ghprb.extensions.status.GhprbSimpleStatus.createCommitStatus(GhprbSimpleStatus.java:308)
	at org.jenkinsci.plugins.ghprb.extensions.status.GhprbSimpleStatus.onBuildComplete(GhprbSimpleStatus.java:267)
	at org.jenkinsci.plugins.ghprb.GhprbBuilds.onCompleted(GhprbBuilds.java:202)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:26)
	at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:211)
	at org.jenkinsci.plugins.workflow.job.WorkflowRun.finish(WorkflowRun.java:664)
	at org.jenkinsci.plugins.workflow.job.WorkflowRun.access$1400(WorkflowRun.java:135)
	at org.jenkinsci.plugins.workflow.job.WorkflowRun$GraphL.onNewHead(WorkflowRun.java:967)
	at org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.notifyListeners(CpsFlowExecution.java:1212)
	at org.jenkinsci.plugins.workflow.cps.CpsThreadGroup$3.run(CpsThreadGroup.java:407)
	at org.jenkinsci.plugins.workflow.cps.CpsVmExecutorService$1.run(CpsVmExecutorService.java:35)
	at hudson.remoting.SingleLaneExecutorService$1.run(SingleLaneExecutorService.java:112)
	at jenkins.util.ContextResettingExecutorService$1.run(ContextResettingExecutorService.java:28)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.kohsuke.github.HttpException: Server returned HTTP response code: 502, message: 'Bad Gateway' for URL: https://api.github.com/repos/dotnet/corefx/statuses/e533df3e30461c2efbb68b7e0b41e49d18bbbed5
	at org.kohsuke.github.Requester.parse(Requester.java:633)
	at org.kohsuke.github.Requester.parse(Requester.java:594)
	at org.kohsuke.github.Requester._to(Requester.java:272)
	... 20 more
Caused by: java.io.IOException: Server returned HTTP response code: 502 for URL: https://api.github.com/repos/dotnet/corefx/statuses/e533df3e30461c2efbb68b7e0b41e49d18bbbed5
	at sun.reflect.GeneratedConstructorAccessor153.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:1944)
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:1939)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:1938)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1508)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at org.kohsuke.github.Requester.parse(Requester.java:612)
	... 22 more
Caused by: java.io.IOException: Server returned HTTP response code: 502 for URL: https://api.github.com/repos/dotnet/corefx/statuses/e533df3e30461c2efbb68b7e0b41e49d18bbbed5
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1894)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:480)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:347)
	at org.kohsuke.github.Requester.parse(Requester.java:602)
	... 22 more
Finished: SUCCESS
```

File: src/System.Drawing.Common/tests/BitmapTests.cs [hunk 1]
```diff
@@ -955,6 +955,7 @@ public void MakeTransparent_Disposed_ThrowsArgumentException()
 
         [ActiveIssue(20884, TestPlatforms.AnyUnix)]
         [ActiveIssue(21886, TargetFrameworkMonikers.NetFramework)]
+        [ActiveIssue(29858)]
```
Comment (stephentoub) [inline src/System.Drawing.Common/tests/BitmapTests.cs]: Should this ActiveIssue be specific to Windows?

File: src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs [hunk 2]
```diff
@@ -24,7 +25,11 @@ public void DumpRuntimeInformationToConsole()
             string lcr = PlatformDetection.LibcRelease;
             string lcv = PlatformDetection.LibcVersion;
 
-            Console.WriteLine($@"{dvs} OS={osd} OSVer={osv} OSArch={osa} Arch={pra} Framework={frd} LibcRelease={lcr} LibcVersion={lcv}");
+            Console.WriteLine($@"### CONFIGURATION: {dvs} OS={osd} OSVer={osv} OSArch={osa} Arch={pra} Framework={frd} LibcRelease={lcr} LibcVersion={lcv}");
+
+            Console.WriteLine($"### BINARIES: {Path.GetDirectoryName(typeof(object).Assembly.Location)} (drive {new DriveInfo(Path.GetDirectoryName(typeof(object).Assembly.Location)).DriveFormat})");
+            Console.WriteLine($"### TEMP PATH: {Path.GetTempPath()} (drive {new DriveInfo(Path.GetTempPath()).DriveFormat})");
+            Console.WriteLine($"### CURRENT DIRECTORY: {Environment.CurrentDirectory}");
```
Comment (stephentoub) [inline src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs]: Looks like this is going to conflict with https://github.com/dotnet/corefx/pull/29816/files#diff-2096cc325fd0971f6ab660d3053eddf9R30

Comment (safern) [reply src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs]: @danmosemsft maybe I can revert this change here and you could pickup this changes in your other PR?

Comment (danmosemsft) [reply src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs]: Sure

**Pull request merged**